content/How_To/environment/Skybox.md

```diff
@@ -10,29 +10,50 @@ video-overview:
 video-content:
 ---
 
-## How To Use a Skybox
-A simulated sky can be added to a scene using a skybox. This is created using a large standard box, special reflective textures and a group of six images, one for each face of the cube.
+## About Skyboxes
+A simulated sky can be added to a scene using a "skybox" ([wikipedia](https://en.wikipedia.org/wiki/Skybox_(video_games))). A skybox is a large standard cube surrounding the scene, with a sky image painted on each face. (Images are a lot easier and faster to render than 3D objects, and just as good for far-distant scenery.)
 
-## Sky Images
-Six jpeg images are necessary to create a skybox, all of them should be square and of the same size which is best as a power of 2, eg 1024px x 1024px.
-The name for each image should have a common part followed by a position given by _px, _nx, _py, _ny, _pz or _nz corresponding whether 
-it is on the positive (p) or negative (n) x, y or z axis.
+## Creating a Cube Texture
+Skybox images are typically loaded with [CubeTexture](/typedoc/classes/babylon.cubetexture). The [constructor](/typedoc/classes/babylon.cubetexture#constructor) takes a base URL and (by default) appends "\_px.jpg", "\_nx.jpg", "\_py.jpg", "\_ny.jpg", "\_pz.jpg" and "\_nz.jpg" to load the +x, -x, +y, -y, +z, and -z facing sides of the cube. (These suffixes may be customized if needed.)
 
-Example Images:
+![Diagram of X/Y/Z axes and CubeTexture sides](/img/how_to/Materials/cubetexture1.png)
 
-![Skybox](/img/how_to/skybox.png)
+Note, despite being a "Texture", CubeTexture can ONLY be used with [reflectionTexture and refractionTexture](/materials/using/reflectionTexture), NOT other Material properties like diffuseTexture. [See below](#makingtheskybox) for the appropriate settings for a skybox.
+
+## Making or Finding Skybox Images
+This is an example set of skybox images:
+
+|<img src="/img/getstarted/skybox_px.jpg" width="100" height="100" alt="some clouds"/>|<img src="/img/getstarted/skybox_nx.jpg" width="100" height="100" alt="more clouds"/>|<img src="/img/getstarted/skybox_py.jpg" width="100" height="100" alt="the sun overhead"/>|<img src="/img/getstarted/skybox_ny.jpg" width="100" height="100" alt="solid gray"/>|<img src="/img/getstarted/skybox_pz.jpg" width="100" height="100" alt="more clouds"/>|<img src="/img/getstarted/skybox_nz.jpg" width="100" height="100" alt="more clouds"/>|
+|:---:|:---:|:---:|:---:|:---:|:---:|
+|skybox\_px.jpg|skybox\_nx.jpg|skybox\_py.jpg|skybox\_nx.jpg|skybox\_pz.jpg|skybox\_nx.jpg|
+<br/>
 
 Notice that the images match seamlessly at the edges of the box:
 
-![Seamless Skybox](/img/how_to/skybox1.png)
+<!-- use HTML to get padding-free table -->

```

Comment:
There's some table support, but do you know how to get rid of the spacing between cells so that I can illustrate the seamlessness of the images? Here's what I want it to look like:

![image](https://user-images.githubusercontent.com/279819/119244316-35c12800-bb24-11eb-9162-64ed53bba363.png)

I don't know how to do this in pure Markdown? (As far as I can tell, the responsive-table support seems to be in CSS, and is independent of whether the table is composed in markdown or HTML. But that's independent.) The previous content used a single image that included the labels, but the labels were hard to read, and of course not searchable as text.

Reply:
@RaananW can provide guidance on this.

Reply:
In this case it does make sense to use HTML, but we need to be sure we are not breaking responsiveness and mobile support.

This is the reasoning behind not using html tables. using markdown tables does that for you automatically. Please add the class `responsiveTable` to the table element.

Reply:
Happy to add that tag, _but_ it's added automatically already to any \<table\> element, including manually created ones:

![image](https://user-images.githubusercontent.com/279819/121787401-b949c400-cb7a-11eb-9e13-f54506ab8234.png)

This is because components/markdownComponents/markdownComponents.tsx maps "table" to Table, which is imported from "react-super-responsive-table"; those mappings apply to manually entered tags as well as markdown-generated ones. The \<Table\> component adds `responsiveTable` to the classes of the HTML table it emits.

So, I don't think it needs to be added, it's automatically consistent?

Reply:
sure, it seems like mdx is taking care of this for us. Thanks!